server/game/cards/03_TWI/units/OutspokenRepresentative.ts

```diff
@@ -0,0 +1,26 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { KeywordName, Trait } from '../../../core/Constants';
+
+export default class OutspokenRepresentative extends NonLeaderUnitCard {
+    protected override getImplementationId () {
+        return {
+            id: '8919416985',
+            internalName: 'outspoken-representative'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addConstantAbility({
+            title: 'While you control another Republic unit, this unit gains Sentinel.',
+            condition: (context) => context.source.controller.isTraitInPlay(Trait.Republic, context.source),
+            ongoingEffect: AbilityHelper.ongoingEffects.gainKeyword({ keyword: KeywordName.Sentinel })
+        });
+        this.addWhenDefeatedAbility({
```

Comment:
nit: our convention is to put a newline between multiple ability definitions for readability